src/main/java/com/example/document/service/DocumentService.java

```diff
@@ -53,37 +55,58 @@ public List<DocumentEntryDTO> uploadExcelDocument(MultipartFile file) throws Exc
             docEntryList.add(newDocumentEntry);
         }
         documentRepository.saveAll(docEntryList);
-        return documentEntryMapper.entityToDto(docEntryList);
+        List<DocumentEntryDTO> documentEntryDTOS = documentEntryMapper.entityToDto(docEntryList);
+
+        List<EmployeeDTO> employeeDTOs = IntStream.range(0, documentEntryDTOS.size())
+                .mapToObj(i -> employeeClient.getEmployeeById(documentEntryDTOS.get(i).getEmployeeId()))
+                .collect(Collectors.toList());
+        return IntStream.range(0, documentEntryDTOS.size())
+                .mapToObj(i -> DocumentWithEmployeeDTO.of(documentEntryDTOS.get(i), employeeDTOs.get(i))).collect(Collectors.toList());
```

Comment:
Let's replace these streams with more readable and short one
The one which is used in insertDocumentEntries